akka-actor/src/main/scala/akka/Done.scala

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (C) 2009-2016 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka
+
+/**
+ * Typically used together with ```Future``` to signal completion
+ * but there is no actual value completed. More clearly signals intent
+ * than ```Unit``` and is available both from Scala and Java (which ```Unit``` is not).
+ */
+sealed abstract class Done
+
+case object Done extends Done {
+  def getInstance() = this
+}
```

Comment:
Technically Java people can now `new Done()`. 
Did we consider `enum`? What are the downsides?